### PR TITLE
UIIN-2668: Jest/RTL: Cover InstancePlugin component with unit tests

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIIN-2668
We added tests for `InstancePlugin`
image

### CODE REVIEW

File: src/components/InstancePlugin/InstancePlugin.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import '../../../test/jest/__mock__';
```
Comment (artem-blazhko) [inline src/components/InstancePlugin/InstancePlugin.test.js:1]: Could you remove this import?

File: src/components/InstancePlugin/InstancePlugin.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { renderWithIntl } from '../../../test/jest/helpers';
+
+const onSelectMock = jest.fn();
+
+describe('InstancePlugin', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should InstancePlugin', () => {
```
Comment (artem-blazhko) [inline src/components/InstancePlugin/InstancePlugin.test.js:13]: What does it mean `'should InstancePlugin'`?
Maybe you meant something like `'should render InstancePlugin'`?

File: src/components/InstancePlugin/InstancePlugin.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    renderWithIntl(<InstancePlugin onSelectMock={onSelectMock} isDisabled={false} />);
```
Comment (artem-blazhko) [inline src/components/InstancePlugin/InstancePlugin.test.js:14]: I think `onSelectMock` property is not expected.
Maybe you meant 
```suggestion
    renderWithIntl(<InstancePlugin onSelect={onSelectMock} isDisabled={false} />);
```

File: src/components/InstancePlugin/InstancePlugin.test.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    expect(screen.getByText(/findInstancePluginNotFound/i)).toBeInTheDocument();
+  });
+
+  it('should InstancePlugin with is disabled prop', () => {
+    renderWithIntl(<InstancePlugin onSelectMock={onSelectMock} isDisabled />);
```
Comment (artem-blazhko) [inline src/components/InstancePlugin/InstancePlugin.test.js:20]:
```suggestion
    renderWithIntl(<InstancePlugin onSelect={onSelectMock} isDisabled />);
```

File: src/components/InstancePlugin/InstancePlugin.test.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    expect(screen.getByText(/findInstancePluginNotFound/i)).toBeInTheDocument();
+  });
+});
```
Comment (artem-blazhko) [inline src/components/InstancePlugin/InstancePlugin.test.js:24]: I think your second test is redundant.
You do not check any enabled and/or disabled elements on the page after component rendering. You just pass different properties that do not have impact on any elements.

**Review by artem-blazhko — CHANGES_REQUESTED** [review]

File: src/components/InstancePlugin/InstancePlugin.test.js (hunk 5 above)
Comment (vashjs) [reply src/components/InstancePlugin/InstancePlugin.test.js:18]: Hi @artem-blazhko, agree, I've removed

cc @UladzislauKutarkin

File: src/components/InstancePlugin/InstancePlugin.test.js (hunk 4 above)
Comment (vashjs) [reply src/components/InstancePlugin/InstancePlugin.test.js]: Hi @artem-blazhko, fixed

cc @UladzislauKutarkin

File: src/components/InstancePlugin/InstancePlugin.test.js (hunk 3 above)
Comment (vashjs) [reply src/components/InstancePlugin/InstancePlugin.test.js]: Hi @artem-blazhko, fixed

cc @UladzislauKutarkin

File: src/components/InstancePlugin/InstancePlugin.test.js (hunk 2 above)
Comment (vashjs) [reply src/components/InstancePlugin/InstancePlugin.test.js]: Fixed

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**